cypress/module/testWebTable.js

```diff
@@ -0,0 +1,78 @@
+import UserAddForm from '../pages/pageWebTables';
+import {USER_DATA, EDIT_USER_DATA, getCellTextAsArray} from '../pages/testData';
```

Comment:
getCellTextAsArray is not exist, delete or create it

Reply:
done